src/tableData/AdapterData/AdapterGoData.js

```diff
@@ -398,6 +398,24 @@ export const AdapterGoData = [
       "For [fs.FS](https://pkg.go.dev/io/fs) and [embed.FS](https://pkg.go.dev/embed)",
     image: "/img/ecosystem/file.png",
   },
+  {
+    title:
+      "[NATS JetStream Adapter](https://github.com/grepplabs/casbin-jetstream)",
+    type: "KV store",
+    author: "[@everesio](https://github.com/everesio)",
+    autoSave: "✅",
+    description: "For [NATS JetStream](https://docs.nats.io/nats-concepts/jetstream)",
+    image: "/img/ecosystem/nats.png",
+  },
+  {
+    title:
+      "[Kubernetes Adapter](https://github.com/grepplabs/casbin-kube)",
+    type: "CRD",
```

Comment:
Use existing types

Reply:
@hsluoyz  Which type should it be? None of the types from the list reflect it.
In casbin-kube, the policies are stored as Kubernetes custom resources.
See: https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/

This is neither NoSQL, nor file-based, nor key-value.